js/main.js

```diff
@@ -432,19 +433,23 @@ $elmPreviewList.addEventListener('click', editClicked);
 function editingChoices() {
   for (let k = 0; k < data.weatherOptions.length; k++) {
     if (data.template[data.weatherOptions[k]] === true) {
-      $weatherChoicesList.children[k].children[0].children[0].className = 'far fa-check-circle';
+      $weatherChoicesList.children[k].querySelector('.far').classList.toggle('fa-check-circle', true);
+      $weatherChoicesList.children[k].querySelector('.far').classList.toggle('fa-circle', false);
     } else if (data.template[data.weatherOptions[k]] === false) {
-      $weatherChoicesList.children[k].children[0].children[0].className = 'far fa-circle';
+      $weatherChoicesList.children[k].querySelector('.far').classList.toggle('fa-check-circle', false);
+      $weatherChoicesList.children[k].querySelector('.far').classList.toggle('fa-circle', true);
     }
   }
 }
 
 function primaryClicked(event) {
-  if (event.target && event.target.nodeName === 'I' && (event.target.className === 'fas fa-star' || event.target.className === 'far fa-star')) {
+  if (event.target && event.target.nodeName === 'I' && event.target.classList.contains('fa-star')) {
     const $elmEntry = event.target.closest('li');
     if (event.target.className === 'far fa-star') {
       for (let priLoop = 0; priLoop < $elmPreviewList.children.length - 1; priLoop++) {
-        $elmPreviewList.children[priLoop].children[0].children[0].className = 'far fa-star';
+        const $elmListing = $elmPreviewList.children[priLoop].querySelector('.fa-star');
+        $elmListing.classList.toggle('far', true);
+        $elmListing.classList.toggle('fas', false);
```

Comment:
Ok, now we can refine this even more. Since you want all of the stars from inside of the preview list, you can use querySelectorAll

```js
const $stars = $elmPreviewList.querySelectorAll('.fa-star');
// loop through just the stars
```